### PR TITLE
Deployment: V3 Hook examples

### PR DESCRIPTION
**Pull request by jubeira**
# Deployment: `V3 Hook examples`

## Deployment tasks checklist:

- [x] Contract(s) deployed to all target networks / testnets
- [x] TX IDs are properly generated
- N/A Action IDs are generated after deployment (if applicable) 
- [x] Outputs for all target networks are generated, and linked in the task `readme`
- [x] Addresses files are up to date with the new deployment 
- [x] Contracts are verified in every network 
- [x] Deployments `CHANGELOG` is updated
- [x] There are no code changes of any kind

## Issue Resolution

N/A

### CODE REVIEW

**Review by EndymionJkb — COMMENTED** [review]
Confirmed that contracts are deployed and verified. Just need to add the outputs to the task readme

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 - Deployed V3 Composite Liquidity Router to Mainnet, Gnosis and Sepolia.
 - Deployed V3 Buffer Router to Mainnet, Gnosis and Sepolia.
 - Deployed V3 Vault Explorer to Mainnet, Gnosis and Sepolia.
+- Deployed V3 Hook Examples Sepolia.
```
Comment (EndymionJkb) [inline CHANGELOG.md:19]:
```suggestion
- Deployed V3 Hook Examples to Sepolia.
```

**Review by joaobrunoah — APPROVED** [review]
LGTM!

**Review by EndymionJkb — APPROVED** [review]
LGTM!

**Pull request merged**